src/js/clickable-component.js

```diff
@@ -202,52 +186,30 @@ class ClickableComponent extends Component {
   handleClick(event) {}
 
   /**
-   * This gets called when a `ClickableComponent` gains focus via a `focus` event.
-   * Turns on listening for `keydown` events. When they happen it
-   * calls `this.handleKeyPress`.
+   * Event handler that is called when a `ClickableComponent` receives a
+   * `keydown` event.
    *
-   * @param {EventTarget~Event} event
-   *        The `focus` event that caused this function to be called.
-   *
-   * @listens focus
-   */
-  handleFocus(event) {
-    Events.on(document, 'keydown', Fn.bind(this, this.handleKeyPress));
-  }
-
-  /**
-   * Called when this ClickableComponent has focus and a key gets pressed down. By
-   * default it will call `this.handleClick` when the key is space or enter.
+   * By default, if the key is Space or Enter, it will trigger a `click` event.
    *
    * @param {EventTarget~Event} event
    *        The `keydown` event that caused this function to be called.
    *
    * @listens keydown
    */
-  handleKeyPress(event) {
-    // Support Space or Enter key operation to fire a click event
+  handleKeyDown(event) {
+
+    // Support Space or Enter key operation to fire a click event. Also,
+    // prevent the event from propagating through the DOM and triggering
+    // Player hotkeys.
     if (keycode.isEventKey(event, 'Space') || keycode.isEventKey(event, 'Enter')) {
-      event.preventDefault();
```

Comment:
we want to call `preventDefault` as well. It will prevent stuff like going page down when hitting the spacebar.

Reply:
Good point. Will update.